Web Quiz Engine (Kotlin)/task/src/engine/WebQuizApiController.kt

```diff
@@ -12,13 +12,14 @@ import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.*
 
-@RestController
+@RestController("/api") // fails hyperskill test for no clear reason
```

Comment:
It's my mistake.
It should be:

```
@RestController
@RequestMapping("/api")
```

Moreover, you should take the habit of testing with Postman when you do these types of changes, you would have seen the reason why it failed immediately.

Reply:
I see. Point taken I will do a proper Postman setup before submitting my next pr as I relied too much on the hyperskill tests.

Reply:
Done